sdcm/cluster.py

```diff
@@ -1693,6 +1693,16 @@ def config_setup(self,
         if debug_install and self.distro.is_rhel_like:
             self.remoter.sudo("yum install -y scylla-gdb", verbose=True, ignore_status=True)
 
+        if self.init_system == "systemd":
```

Comment:
how do you make sure that his code will run on Debian 10 and Ubuntu 20.04 only?

Reply:
please add a check that systemd version is greater than 240

Reply:
sure

Reply:
done